code/pages/ExtensibleSearchPage.php

```diff
@@ -551,6 +551,13 @@ public function searchableTypes($default = null) {
 		return $listType;
 	}
 
+	public function SearchOverlay() {
+		$suggestions = $this->Suggestions()->filter(array(
+			'Approved' => 1,
+		))->sort('Frequency', 'DESC')->limit(5);
+
+		return $suggestions->renderWith(array('ExtensibleSearch_overlay'));
```

Comment:
I feel like this could be implemented a little cleaner. Rather than calling this function from a template, could you <% include %> the overlay template, which attempts to retrieve the search suggestions? That way you could also split the search suggestion retrieval out into a function that it's possible to use elsewhere.